data/zilliqa/assets/zil1hau7z6rjltvjc95pphwj57umdpvv0d6kh2t8zk/meta.json

```diff
@@ -5,14 +5,17 @@
   "supply": "10000000",
   "email": "contact@carbontoken.info",
   "whitepaper": "https://github.com/GenesysLabs/whitepapers",
+  "holders": true,
```

Comment:
Can you remove this one? I'm thinking of reworking that, since not many people update it, maybe with an automated viewblock process, tbd

Reply:
Ok. Am closing this.. will make changes and pull again. I agree automated might be good here. It can be easily manipulated as well though. So not sure how valid it is going forward.